reference-service/src/test/resources/db/migration/common/V3.81__create_funding_reason_table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `FundingReason` (
+  `id` varchar(36) NOT NULL,
+  `reason` varchar(255) NOT NULL,
+  `status` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO FundingReason (`id`, `reason`, `status`) VALUES
+(uuid(), 'Long Term Plan', 'CURRENT'),
+(uuid(), 'Redistribution', 'CURRENT'),
+(uuid(), 'Expansion Posts', 'CURRENT'),
+(uuid(), 'Cancer and Diagnostics', 'CURRENT');
```

Comment:
This script is exactly the same with the one under /java/resource, so I guess this one is redundant.
Could you try removing this file and reruning the tests please?

Reply:
`Table "FUNDINGREASON" not found; SQL statement:`

This is what I originally saw locally (and also [e578712](https://github.com/Health-Education-England/TIS-REFERENCE/pull/350/commits/e578712d1c02e67cb4a90280945e3d587d9fdac6) after removal) as the tests are all integration tests against a H2 db

Reply:
I've just tried with the script on the PR: https://github.com/Health-Education-England/TIS-REFERENCE/pull/349/files, and this script in the test became redundant.

Reply:
Ah, perhaps after 349 is merged it will work then?

Reply:
Yes, after a rebase, it will work as the script is not on this PR now.
Btw, don't forget the version bump.